charmstore_test.go

```diff
@@ -496,7 +496,7 @@ func (s *charmStoreRepoSuite) TestLatest(c *gc.C) {
 	// Run the tests.
 	for i, test := range tests {
 		c.Logf("test %d: %s", i, test.about)
-		revs, err := s.repo.Latest(test.urls...)
+		revs, err := s.repo.(*charmrepo.CharmStore).Latest(test.urls...)
```

Comment:
We would still need a type coercion somewhere as NewCharmStore() returns the interface. I'd prefer to do it here and have the rest of the tests work with the interface.